debian/control

```diff
@@ -5,8 +5,9 @@ Maintainer: Rob Smith <kormoc@gmail.com>
 Homepage: https://github.com/BrightcoveOS/Diamond
 Vcs-Git: git://github.com/BrightcoveOS/Diamond.git
 Vcs-Browser: https://github.com/BrightcoveOS/Diamond
-Build-Depends: debhelper (>= 7), python (>= 2.4), python-support, python-mock, python-configobj, cdbs
+Build-Depends: debhelper (>= 7), python-all, python-setuptools, dh-python, python-mock, python-configobj, cdbs
```

Comment:
Can we instead do:

```
python-support | dh-python
```

That would allow us to support older versions I think (assuming dh-python isn't there).